package.json

```diff
@@ -60,7 +60,7 @@
     "jest": ">= 20"
   },
   "engines": {
-    "node": ">= 4"
+    "node": ">= 8"
```

Comment:
Since this will be a breaking change, I think it'd be better to upgrade to the latest nock (v13), which requires node 10, i.e., the minimal Node.js LTS version at the moment.